packages/flame/src/Core/index.tsx

```diff
@@ -145,13 +136,4 @@ const FlameGlobalStyles: React.FunctionComponent<{ themeName?: string }> = ({ th
   );
 };
 
-export {
-  lightTheme,
-  flameTheme,
-  FlameFonts,
-  FlameTheme,
-  FlameGlobalStyles,
-  themeGet,
-  ThemeUIFlame,
-  border,
-};
+export { lightTheme, flameTheme, themeGet, ThemeUIFlame };
```

Comment:
Why go with a mix of both exporting inline and at the end of the file?

Reply:
Debugging shenanigans with storybook + their react-docgen-typescript, gonna revert the changes